crates/re_renderer/src/wgpu_resources/mod.rs

```diff
@@ -115,6 +115,7 @@ impl WgpuResourcePools {
     }
 }
 
+#[derive(Copy, Clone)]
 pub struct TextureRowDataInfo {
```

Comment:
It may be worth noting that the padding always come at the _end_ of the row.

Reply:
kk, already accepted this as fact of life but I can see how that's non-obvious